device-types/Teltonika/RUTX09.yaml

```diff
@@ -0,0 +1,34 @@
+---
+manufacturer: Teltonika
+model: RUTX09
+slug: teltonika-rutx09
+part_number: RUTX09
+u_height: 0.5
+is_full_depth: false
+weight: 455
+weight_unit: g
+comments: '[RUTX09 Specs](https://teltonika-networks.com/cdn/products/2023/01/63b7f7745a32a6-87050095/rutx09-datasheet-v12.pdf)'
+power-ports:
+  - name: PS0
+    type: dc-terminal
+    maximum_draw: 9
+  - name: USB_1
+    type: usb-a
```

Comment:
According to the doc this is not for power. It says the USB applications are "Samba share, USB-to-serial"